python-sdk/src/astro/databases/snowflake.py

```diff
@@ -480,30 +481,16 @@ def create_table_using_schema_autodetection(
         table: BaseTable,
         file: File | None = None,
         dataframe: pd.DataFrame | None = None,
-        columns_names_capitalization: ColumnCapitalization = "lower",
+        columns_names_capitalization: ColumnCapitalization = "original",
     ) -> None:
         """
-        Create a SQL table, automatically inferring the schema using the given file.
+        Dummy function. We are inferring schema and creating table in `load_pandas_dataframe_to_table`.
```

Comment:
Why are we defining this function at all? Perhaps we shouldn't define and just use the parent class implementation.

Reply:
We want to override the default implementation. Since prep_table() is not allowing us to add quotes. we want to skip it and use a single method(pandas_tool.write_write_pandas()) to create and populate a snowflake table.